model-optimizer/extensions/load/onnx/loader.py

```diff
@@ -47,6 +48,7 @@ def load(self, graph: Graph):
             ) from e
         log.debug("Number of nodes in NX graph: {}".format(graph.number_of_nodes()))
 
+        send_op_names_info('onnx', graph)
```

Comment:
For models from ONNX and TF we read nodes in the sub-graph (for example, in operation If (not yet merged) and Loop) in the dedicated extractor. For example, `model-optimizer/extensions/front/tf/while_ext.py` and `model-optimizer/extensions/front/onnx/loop_ext.py`. So we need to collect information about graph op statistics in the `graph` object attributes, update it with information from the sub-graph (Loop, If) and only then send it.

Reply:
Implemented op names saving to graph field "op_names_statistics" during model loading and in while_ext, loop_ext. And moved op names telemetry sending to the end of model loading after extractors launch.